supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -55,6 +55,16 @@ $ oc describe node __<node_name>__
 ----
 ====
 
+When applying multiple attributes to a block, use the following syntax:
```

Comment:
I meant to leave a comment here but apparently forgot to save it. I think that this would probably be more appropriate to be added to the [AsciiDoc guide](https://redhat-documentation.github.io/asciidoc-markup-conventions/). wdyt?

Reply:
Maybe? I've trimmed down the example so it's not so beefy now, and it would be nice for the tip to be in a place where it's probably pretty likely to come up (not to mention there's already an example syntax right above it. Are you suggesting another entry added to the big table specifically about combining attributes?

Reply:
I'm on the fence about this. Maybe if it was wrapped as a TIP admonition?

I'll defer to others on the @redhat-documentation/ccs-style-council if they think it's okay to add this tip here.

Reply:
If it's a general tip rather than a directive maybe the AsciiDoc guide is the best place ... keeping the focus here on directives would be more in line with minimalism principles. I don't think we currently have lots of TIPs included.

It's a small one though so wouldn't create a big problem to have it here with a TIP admonition either.

Reply:
@bergerhoffer @ndeevy I've updated it as a TIP admonition block.

Reply:
@adellape @bergerhoffer I'm confused about what you're trying to say here...is the tip to use `[source,terminal,subs="+quotes"]` for code blocks that include user-replaceable values? If yes, then I think this will add more confusion...or maybe add a link to detailed info on how to use these attributes or whatever you call them for code blocks. I'm not sure about other writers, but I don't have a real great grasp on all the different options available for code blocks...I tend to reuse what I already see for a similar usage in the doc I'm working on.